cypress/support/commands.js

```diff
@@ -480,3 +480,30 @@ Cypress.Commands.add("verifyMobileFieldAndDisplayedValue", (mobileNumber) => {
 			expect(displayedValue).to.equal(mobileNumber);
 		});
 });
+
+Cypress.Commands.add("enterEmailVerificationCodeForEventUser", () => {
```

Comment:
Instead of duplicating this method could you not change `enterEmailVerificationCode()` and pass it in the email address?

Reply:
Resolved